resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/shuffle-service/Dockerfile

```diff
@@ -15,25 +15,12 @@
 # limitations under the License.
 #
 
-FROM openjdk:8-alpine
+FROM spark-base
```

Comment:
I think we want `spark-kubernetes` because we might namespace conflict with Nomad in particular, which is another effort being done to run the driver and executors in Docker containers.